modules/swagger-jaxrs2/src/main/java/io/swagger/jaxrs2/Reader.java

```diff
@@ -46,389 +46,401 @@
+        for (io.swagger.oas.annotations.Operation callbackOperation : apiCallback.operation()) {
+            Operation callbackNewOperation = new Operation();
+            setOperationObjectFromApiOperationAnnotation(callbackNewOperation, callbackOperation);
+            setPathItemOperation(pathItemObject, callbackOperation.method(), callbackNewOperation);
+        }
+
+        callbackObject.addPathItem(apiCallback.callbackUrlExpression(), pathItemObject);
+        callbackMap.put(apiCallback.name(), callbackObject);
+
+        return callbackMap;
+    }
+
+    private void setPathItemOperation(PathItem pathItemObject, String method, Operation callbackNewOperation) {
+        switch (method) {
+            case POST_METHOD:
+                pathItemObject.post(callbackNewOperation);
+                break;
+            case GET_METHOD:
+                pathItemObject.get(callbackNewOperation);
+                break;
+            case DELETE_METHOD:
+                pathItemObject.delete(callbackNewOperation);
+                break;
+            case PUT_METHOD:
+                pathItemObject.put(callbackNewOperation);
+                break;
+            case PATCH_METHOD:
+                pathItemObject.patch(callbackNewOperation);
+                break;
+            case TRACE_METHOD:
+                pathItemObject.trace(callbackNewOperation);
+                break;
+            case HEAD_METHOD:
+                pathItemObject.head(callbackNewOperation);
+                break;
+            case OPTIONS_METHOD:
+                pathItemObject.options(callbackNewOperation);
+                break;
+            default:
+                // Do nothing here
+                break;
+        }
+    }
+
+    private void setOperationObjectFromApiOperationAnnotation(Operation operation, io.swagger.oas.annotations.Operation apiOperation) {
+        if (StringUtils.isNotBlank(apiOperation.summary())) {
+            operation.setSummary(apiOperation.summary());
+        }
+        if (StringUtils.isNotBlank(apiOperation.description())) {
+            operation.setDescription(apiOperation.description());
+        }
+        if (StringUtils.isNotBlank(apiOperation.operationId())) {
+            operation.setOperationId(getOperationId(apiOperation.operationId()));
+        }
+        if (apiOperation.deprecated()) {
+            operation.setDeprecated(apiOperation.deprecated());
+        }
+        ReaderUtils.getStringListFromStringArray(apiOperation.tags()).ifPresent(operation::setTags);
+        OperationParser.getTags(apiOperation.tags()).ifPresent(tag -> openApiTags.addAll(tag));
+        OperationParser.getExternalDocumentation(apiOperation.externalDocs()).ifPresent(operation::setExternalDocs);
+        OperationParser.getRequestBody(apiOperation.requestBody()).ifPresent(operation::setRequestBody);
+        OperationParser.getApiResponses(apiOperation.responses()).ifPresent(operation::setResponses);
```

Comment:
Do a check to see if `responses.size()` is > 0 or not.  The `apiOperation.responses()` is never `Empty` because it's an empty map.  That will avoid this:

```
        "responses": {
          "default": {}
        }
```
Which we don't want